pkg/cmd/build/logger.go

```diff
@@ -54,7 +62,7 @@ func deployDisplayer(ctx context.Context, ch chan *client.SolveStatus) error {
 					oktetoLog.Info(err.Error())
 					continue
 				}
-				t.display()
+				t.display(outputMode)
```

Comment:
```suggestion
				t.display(o.OutputMode)
```

can we use directly `o.OutputMode`?

Reply:
Yes, we can. I had this thought at the start while making changes. However, I felt the code would be more readable and understandable if we create a variable rather than directly passing value. Do you want to pass the value directly?

Reply:
if you think is more readable your approach go ahead with it 👍